.circleci/config.yml

```diff
@@ -99,6 +99,47 @@ commands:
             python ./test_runner.py --suite=push -o $ENZOTEST_DIR --answer-name=$ANSWER_NAME --local --strict=high --verbose << parameters.flags >>
           fi
 
+  compile-test:
+    description: "Test that enzo compiles in a few different configurations."
+    steps:
+      - run: |
+          source $BASH_ENV
+          source $HOME/venv/bin/activate
+          ./configure
+          cd src/enzo
+          make machine-linux-gnu
+          make load-config-test-suite
+          # test various baryon/particle precisions
+          for prec in 32 64
+          do for part in 32 64 128
+          do
+          if [ $prec == $part ]; then
+          continue
+          fi
+          make precision-$prec
+          make particles-$part
+          make clean
+          make -j 4
+          done ; done
+          # return this to default
+          make particles-64
+          # test various integer/particle-id precisions
+          for inte in 32 64
+          do for pids in 32 64
+          do
+          if [ $inte == $pids ]; then
```

Comment:
Same comment as above

Reply:
It looks like the test-suite config is the following:
```
   CONFIG_PRECISION  [precision-{32,64}]                     : 64
   CONFIG_PARTICLES  [particles-{32,64,128}]                 : 64
   CONFIG_INTEGERS  [integers-{32,64}]                       : 32
   CONFIG_PARTICLE_IDS  [particle-id-{32,64}]                : 32
```
I'll change it to skip that one.